src/state/user/hooks.tsx

```diff
@@ -117,6 +118,15 @@ export function useShowSurveyPopup(): [boolean | undefined, (showPopup: boolean)
   return [showSurveyPopup, toggleShowSurveyPopup]
 }
 
+export function useHideNFTWelcomeModal(): [boolean | undefined, () => void] {
+  const dispatch = useAppDispatch()
+  const hideNFTWelcomeModal = useAppSelector((state) => state.user.hideNFTWelcomeModal)
+  const hideModal = useCallback(() => {
+    dispatch(updateHideNFTWelcomeModal({ hideNFTWelcomeModal: true }))
+  }, [dispatch])
+  return [hideNFTWelcomeModal, hideModal]
+}
```

Comment:
Open to feedback if there's better way to do it! Based on other PRs, but it looks a bit verbose for just a single boolean!

Reply:
i think this is the standard way we do it, so lgtm

Reply:
Thanks! I was referring to your PR actually, but just couldn't bring the number up while writing this comment!